src/config/capabilityStatementResources.json

```diff
@@ -120,6 +120,14 @@
           }],
           "name" : "package",
           "definition" : "http://hl7.org/fhir/us/cqfmeasures/OperationDefinition/Library-package"
+        },
+        {
+          "extension" : [{
+            "url" : "http://hl7.org/fhir/StructureDefinition/capabilitystatement-expectation",
+            "valueCode" : "SHALL"
+          }],
+          "name" : "submit",
+          "definition" : "example.com"
```

Comment:
Yeah I feel like there's a tradeoff here where if we include the description then we may forget to change it to the actual operation definition down the line (if it ever gets created), whereas example.com is more obviously incorrect. But at the same time I see what you're saying with this not giving any information to the capability statement. I'll include this definition and then I'll put a TODO comment in `serverConfig` about how the authoring measure repo definitions don't point to operation definitions so that we remember to change them in the future

Reply:
🤝